main/src/com/google/refine/expr/functions/Cross.java

```diff
@@ -47,33 +47,39 @@
 import com.google.refine.model.Project;
 
 public class Cross implements Function {
-    
+
+    public static final String EVAL_ERROR_MESSAGE =
+            " expects a string or cell, a project name to join with, and a column name in that project. " +
+                    "Optional accepts a regular expression separator for source values.";
+
     @Override
     public Object call(Properties bindings, Object[] args) {
-        if (args.length == 3) {
+        if (args.length >= 3) {
```

Comment:
Calling `cross` with more than 4 arguments should fail with an error, it looks like it currently does not.

Reply:
I saw that also after the revert and agree, let's fail with error on greater than 4 arguments as well as having a test for when there are missing arguments... in other words, when there are arguments 1,2,null,4 it should throw error as we currently do.